SKILL.tsv

```diff
@@ -5440,29 +5440,29 @@ SKILL_20151022_005439	$Increases the damage dealt on an enemy with [Esquive Touc
 SKILL_20151022_005440	$Flanconnade: Enhance
 SKILL_20151022_005441	$Increases the damage dealt on an enemy with [Flanconnade] by 1% per attribute level.
 SKILL_20151022_005442	$Summon Servant: Quick Order
-SKILL_20151022_005443	
+SKILL_20151022_005443	$The servant applies beneficial effects more often.
 SKILL_20151022_005444	$Slow: Nullify Decrease
-SKILL_20151022_005445	
+SKILL_20151022_005445	$Enemies affected by [Slow] get a 6% lower chance per attribute level of nullifying an attack.
```

Comment:
Decreases the chance of nullifying the attack by 6% per attribute level when enemies are affected by [Slow].